src/main/java/com/endava/soa_5/basic/Exercise_3.java

```diff
@@ -0,0 +1,35 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_3 {
+    //Write a method that prints day of the week by given number.
+    String message;
+
+    String setDayForResult(int day) {
```

Comment:
Try to optimize as in the Exercise_2.java

Reply:
Done!